apps/web/modules/signup-view.tsx

```diff
@@ -333,6 +336,19 @@ export default function Signup({
           <HeadSeo title={t("sign_up")} description={t("sign_up")} />
           {/* Left side */}
           <div className="ml-auto mr-auto mt-0 flex w-full max-w-xl flex-col px-4 pt-6 sm:px-16 md:px-20 lg:mt-12 2xl:px-28">
+            {displayBackButton && (
+              <div className="flex w-fit lg:-mt-12">
+                <Button
+                  color="minimal"
+                  className="hover:bg-subtle todesktop:mt-10 [&[aria-current='page']]:bg-emphasis [&[aria-current='page']]:text-emphasis group-hover:text-default text-emphasis group mb-6 flex h-6 max-h-6 w-full flex-row items-center rounded-md px-3 py-2 text-sm font-medium leading-4 transition"
```

Comment:
I see your using the imported `<Button />` component from the UI library. Why are there so many classNames here? Wouldn't this button also match the rest of the buttons we use through out the app?

Reply:
removed unneeded classnames!